app/src/main/java/org/dslul/openboard/inputmethod/latin/LatinIME.java

```diff
@@ -1958,6 +1958,10 @@ public boolean shouldSwitchToOtherInputMethods() {
         return mRichImm.shouldOfferSwitchingToNextInputMethod(token, fallbackValue);
     }
 
+    public boolean shouldShowNumberRowKey() {
+        return mSettings.getCurrent().mShowsNumberRowKey;
+    }
+
```

Comment:
This is not strictly necessary, it's better if you get the pref from KeyboardSwitcher directly